src/pkg/inspection/controller.go

```diff
@@ -331,6 +332,20 @@ func (c *controller) Run(ctx context.Context, policy *v1alpha1.InspectionPolicy)
 		}
 	}
 
+	// Read config from InspectionPolicy, send assessment reports to Governor api if governor enabled.
+	if policy.Spec.Inspection.Assessment.Governor.Enabled {
+		governorConfig := policy.Spec.Inspection.Assessment.Governor
+		exporter := governor.GovernorExporter{
+			Report:    report,
+			ClusterID: governorConfig.ClusterID,
+			ApiURL:    governorConfig.URL,
+			ApiToken:  governorConfig.APIToken,
+		}
```

Comment:
This block can probably out of the loop. The inspection policy data is not going to change.